lib/elixir_sense/core/module_store.ex

```diff
@@ -34,7 +34,8 @@ defmodule ElixirSense.Core.ModuleStore do
     Applications.get_modules_from_applications()
     |> Enum.filter(fn module ->
       try do
-        function_exported?(module, :module_info, 1)
+        Code.ensure_compiled(module)
```

Comment:
I believe we want this to placate dialyzer.
```suggestion
        _ = Code.ensure_compiled(module)
```